src/sp/it/pl/main/App.kt

```diff
@@ -481,17 +463,9 @@ class App: Application(), Configurable<Any> {
         }
     }
 
-    private fun dispose() {
-        Player.dispose()
-        db.stop()
-        Action.stopActionListening()
-        appCommunicator.stop()
-    }
-
     /** Close this app normally. Invokes [stop] as a result. */
```

Comment:
Reworded. The invocation is hidden and done by javaFx toolkit. It knows to close the app because JVM is closed by Platform.exit(). It's not good way, but there were some lose threads from libs before so I decided to use this approach.